File: RSPAClient/Core/AtPar.ViewModel/VM_MT_ATPAR_USER_ADD.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_MT_ATPAR_USER_ADD
    {
        public string MODE { get; set; }
        public string USER_ID { get; set; }
        public bool PASSHASH_REQUIRED { get; set; }
        public string PASSWORD { get; set; }
        public Nullable<int> TOKEN_EXPIRY_PERIOD { get; set; }
        public Nullable<int> IDLE_TIME { get; set; }
        public string FIRST_NAME { get; set; }
        public string LAST_NAME { get; set; }
        public string MIDDLE_INITIAL { get; set; }
        public string EMAIL_ID { get; set; }
        public string PHONE1 { get; set; }
        public string PHONE2 { get; set; }
        public string FAX { get; set; }
        public string PAGER { get; set; }
        public string ORG_GROUP_ID { get; set; }
        public string PROFILE_ID { get; set; }
        public string LDAP_USER { get; set; }
        public string LDAP_ROLE { get; set; }
        public string LDAP_ORG { get; set; }
        public string TIME_RESTRICTIONS { get; set; }
        public string PASSWD_RESET_REQUIRED { get; set; }
        public string LAST_UPDATE_USER { get; set; }
        public bool ACCOUNT_DISABLED { get; set; }
        public string USERDN { get; set; }
        public string ENTERPRISE_SYSTEM { get; set; }
        public long STATUS_CODE { get; set; }
        public string CREATE_USER_ID { get; set; }
        public string SESSIONEXPIRY { get; set; }

    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_MT_ATPAR_USER_ADD.ts

export class VM_MT_ATPAR_USER_ADD {
    public MODE: string;
    public USER_ID: string = '';
    public PASSHASH_REQUIRED: boolean = false;
    public PASSWORD: string='';
    public CPASSWORD: string='';
    public TOKEN_EXPIRY_PERIOD: number;
    public IDLE_TIME: number
    public FIRST_NAME: string='';
    public LAST_NAME: string='';
    public MIDDLE_INITIAL: string='';
    public EMAIL_ID: string='';
    public PHONE1: string='';
    public PHONE2: string='';
    public FAX: string='';
    public PAGER: string='';
    public ORG_GROUP_ID: string='';
    public PROFILE_ID: string='';
    public LDAP_USER: string='';
    public LDAP_ROLE: string='';
    public LDAP_ORG: string='';
    public TIME_RESTRICTIONS: string = '1-12:00,23:59;2-12:00,23:59;3-12:00,23:59;4-12:00,23:59;5-12:00,23:59;6-12:00,23:59;7-12:00,23:59;';
    public PASSWD_RESET_REQUIRED: string = 'N';
    public PWD_RESET_REQUIRED: boolean = false;
    public LAST_UPDATE_USER: string = '';
    public ACCOUNT_DISABLED: boolean = false;
    public USERDN: string='';
    public ENTERPRISE_SYSTEM: string;
    public STATUS_CODE: number;
}
